nid05140:16579:16579 [0] NCCL INFO Bootstrap : Using [0]ipogif0:148.187.52.61<0>
nid05140:16579:16579 [0] NCCL INFO NET/Plugin : No plugin found (libnccl-net.so).
libibverbs: Could not locate libibgni (/usr/lib64/libibgni.so.1: undefined symbol: verbs_uninit_context)
libibverbs: Warning: couldn't open config directory '/opt/cray/rdma-core/22.3-7.0.2.1_2.21__g42f5f32b.ari/etc/libibverbs.d'.
nid05140:16579:16579 [0] NCCL INFO NET/IB : No device found.
nid05140:16579:16579 [0] NCCL INFO NET/Socket : Using [0]ipogif0:148.187.52.61<0>
NCCL version 2.4.8+cuda10.1
nid05140:16579:16612 [0] NCCL INFO Setting affinity for GPU 0 to ffffff
nid05151:1285:1285 [0] NCCL INFO Bootstrap : Using [0]ipogif0:148.187.52.72<0>
nid05141:31213:31213 [0] NCCL INFO Bootstrap : Using [0]ipogif0:148.187.52.62<0>
nid05150:29638:29638 [0] NCCL INFO Bootstrap : Using [0]ipogif0:148.187.52.71<0>
nid05150:29638:29638 [0] NCCL INFO NET/Plugin : No plugin found (libnccl-net.so).
nid05151:1285:1285 [0] NCCL INFO NET/Plugin : No plugin found (libnccl-net.so).
libibverbs: Could not locate libibgni (/usr/lib64/libibgni.so.1: undefined symbol: verbs_uninit_context)
libibverbs: Warning: couldn't open config directory '/opt/cray/rdma-core/22.3-7.0.2.1_2.21__g42f5f32b.ari/etc/libibverbs.d'.
nid05151:1285:1285 [0] NCCL INFO NET/IB : No device found.
nid05151:1285:1285 [0] NCCL INFO NET/Socket : Using [0]ipogif0:148.187.52.72<0>
nid05151:1285:1309 [0] NCCL INFO Setting affinity for GPU 0 to ffffff
nid05141:31213:31213 [0] NCCL INFO NET/Plugin : No plugin found (libnccl-net.so).
libibverbs: Could not locate libibgni (/usr/lib64/libibgni.so.1: undefined symbol: verbs_uninit_context)
libibverbs: Warning: couldn't open config directory '/opt/cray/rdma-core/22.3-7.0.2.1_2.21__g42f5f32b.ari/etc/libibverbs.d'.
nid05141:31213:31213 [0] NCCL INFO NET/IB : No device found.
nid05141:31213:31213 [0] NCCL INFO NET/Socket : Using [0]ipogif0:148.187.52.62<0>
libibverbs: Could not locate libibgni (/usr/lib64/libibgni.so.1: undefined symbol: verbs_uninit_context)
libibverbs: Warning: couldn't open config directory '/opt/cray/rdma-core/22.3-7.0.2.1_2.21__g42f5f32b.ari/etc/libibverbs.d'.
nid05150:29638:29638 [0] NCCL INFO NET/IB : No device found.
nid05150:29638:29638 [0] NCCL INFO NET/Socket : Using [0]ipogif0:148.187.52.71<0>
nid05141:31213:31236 [0] NCCL INFO Setting affinity for GPU 0 to ffffff
nid05150:29638:29661 [0] NCCL INFO Setting affinity for GPU 0 to ffffff
nid05151:1285:1309 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05151:1285:1309 [0] NCCL INFO include/net.h:19 -> 2
nid05151:1285:1309 [0] NCCL INFO CUDA Dev 0[0], Socket NIC distance :  SYS
nid05140:16579:16612 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05140:16579:16612 [0] NCCL INFO include/net.h:19 -> 2
nid05140:16579:16612 [0] NCCL INFO CUDA Dev 0[0], Socket NIC distance :  SYS
nid05150:29638:29661 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05150:29638:29661 [0] NCCL INFO include/net.h:19 -> 2
nid05150:29638:29661 [0] NCCL INFO CUDA Dev 0[0], Socket NIC distance :  SYS
nid05141:31213:31236 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05141:31213:31236 [0] NCCL INFO include/net.h:19 -> 2
nid05141:31213:31236 [0] NCCL INFO CUDA Dev 0[0], Socket NIC distance :  SYS
nid05140:16579:16612 [0] NCCL INFO Channel 00 :    0   1   2   3
nid05150:29638:29661 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05150:29638:29661 [0] NCCL INFO include/net.h:19 -> 2
nid05141:31213:31236 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05151:1285:1309 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05151:1285:1309 [0] NCCL INFO include/net.h:19 -> 2
nid05140:16579:16612 [0] NCCL INFO Could not find real path of /sys/class/net/ipogif0/device
nid05140:16579:16612 [0] NCCL INFO include/net.h:19 -> 2
nid05141:31213:31236 [0] NCCL INFO include/net.h:19 -> 2
nid05141:31213:31236 [0] NCCL INFO Ring 00 : 0 -> 1 [receive] via NET/Socket/0
nid05151:1285:1309 [0] NCCL INFO Ring 00 : 2 -> 3 [receive] via NET/Socket/0
nid05150:29638:29661 [0] NCCL INFO Ring 00 : 1 -> 2 [receive] via NET/Socket/0
nid05141:31213:31236 [0] NCCL INFO NET/Socket: Using 1 threads and 1 sockets per thread
nid05151:1285:1309 [0] NCCL INFO NET/Socket: Using 1 threads and 1 sockets per thread
nid05140:16579:16612 [0] NCCL INFO Ring 00 : 3 -> 0 [receive] via NET/Socket/0
nid05150:29638:29661 [0] NCCL INFO NET/Socket: Using 1 threads and 1 sockets per thread
nid05140:16579:16612 [0] NCCL INFO NET/Socket: Using 1 threads and 1 sockets per thread
nid05141:31213:31236 [0] NCCL INFO Ring 00 : 1 -> 2 [send] via NET/Socket/0
nid05151:1285:1309 [0] NCCL INFO Ring 00 : 3 -> 0 [send] via NET/Socket/0
nid05150:29638:29661 [0] NCCL INFO Ring 00 : 2 -> 3 [send] via NET/Socket/0
nid05140:16579:16612 [0] NCCL INFO Ring 00 : 0 -> 1 [send] via NET/Socket/0
nid05140:16579:16612 [0] NCCL INFO Using 256 threads, Min Comp Cap 6, Trees disabled
nid05151:1285:1309 [0] NCCL INFO comm 0x2aab38002260 rank 3 nranks 4 cudaDev 0 nvmlDev 0 - Init COMPLETE
nid05150:29638:29661 [0] NCCL INFO comm 0x2aab38002260 rank 2 nranks 4 cudaDev 0 nvmlDev 0 - Init COMPLETE
nid05141:31213:31236 [0] NCCL INFO comm 0x2aab38002260 rank 1 nranks 4 cudaDev 0 nvmlDev 0 - Init COMPLETE
nid05140:16579:16612 [0] NCCL INFO comm 0x2aab40002260 rank 0 nranks 4 cudaDev 0 nvmlDev 0 - Init COMPLETE
nid05140:16579:16579 [0] NCCL INFO Launch mode Parallel
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    main()
  File "main.py", line 46, in main
    run(**args_dict)
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 145, in summa_mlp_run
    mlp = SummaMLP(hidden_dim=hidden_dim,
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 45, in __init__
    self.w1 = Parameter(torch.Tensor(self.hidden_dim * 4 // self.dim_col,
RuntimeError: [enforce fail at CPUAllocator.cpp:64] . DefaultCPUAllocator: can't allocate memory: you tried to allocate 524288000000 bytes. Error code 12 (Cannot allocate memory)
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    main()
  File "main.py", line 46, in main
    run(**args_dict)
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 145, in summa_mlp_run
    mlp = SummaMLP(hidden_dim=hidden_dim,
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 45, in __init__
    self.w1 = Parameter(torch.Tensor(self.hidden_dim * 4 // self.dim_col,
RuntimeError: [enforce fail at CPUAllocator.cpp:64] . DefaultCPUAllocator: can't allocate memory: you tried to allocate 524288000000 bytes. Error code 12 (Cannot allocate memory)
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    main()
  File "main.py", line 46, in main
    run(**args_dict)
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 145, in summa_mlp_run
    mlp = SummaMLP(hidden_dim=hidden_dim,
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 45, in __init__
    self.w1 = Parameter(torch.Tensor(self.hidden_dim * 4 // self.dim_col,
RuntimeError: [enforce fail at CPUAllocator.cpp:64] . DefaultCPUAllocator: can't allocate memory: you tried to allocate 524288000000 bytes. Error code 12 (Cannot allocate memory)
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    main()
  File "main.py", line 46, in main
    run(**args_dict)
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 145, in summa_mlp_run
    mlp = SummaMLP(hidden_dim=hidden_dim,
  File "/users/youyang9/lsg/Megatron-LM/scripts/summa/piz_daint/summa_mlp.py", line 45, in __init__
    self.w1 = Parameter(torch.Tensor(self.hidden_dim * 4 // self.dim_col,
RuntimeError: [enforce fail at CPUAllocator.cpp:64] . DefaultCPUAllocator: can't allocate memory: you tried to allocate 524288000000 bytes. Error code 12 (Cannot allocate memory)
srun: error: nid05150: task 2: Exited with exit code 1
srun: Terminating job step 26287302.0
srun: error: nid05151: task 3: Exited with exit code 1
srun: error: nid05141: task 1: Exited with exit code 1
srun: error: nid05140: task 0: Exited with exit code 1


Batch Job Summary Report (version 20.09.1) for Job "summa_pytorch" (26287302) on daint
Job information (1/3)
-----------------------------------------------------------------------------------------------------
             Submit            Eligible               Start                 End    Elapsed Time limit
------------------- ------------------- ------------------- ------------------- ---------- ----------
2020-10-16T07:06:32 2020-10-16T07:06:32 2020-10-16T09:21:21 2020-10-16T09:22:03   00:00:42   00:15:00
-----------------------------------------------------------------------------------------------------

Job information (2/3)
-------------------------------------------------------------
    Username      Account    Partition   NNodes        Energy
------------ ------------ ------------ -------- -------------
    youyang9        s1054       normal        4     11.888 kJ

Job information (3/3) - GPU utilization data
---------------------------------------------------
   Node name       Usage     Max mem Execution time
------------ ----------- ----------- --------------
    nid05151         8 %    10701 MB       00:00:22
    nid05140        11 %    10701 MB       00:00:23

